{"name": "test_user_login_failure[chromium]", "status": "passed", "description": "\n    Test case to verify failed login using invalid credentials.\n    ", "parameters": [{"name": "browser_type", "value": "'chromium'"}], "start": 1727354169194, "stop": 1727354176497, "uuid": "5d7e546c-a840-44ca-a945-8b390e524c6a", "historyId": "df7edcb30dcc084392721104227917d7", "testCaseId": "17d3089194985caa2bc4fdc96dc1b569", "fullName": "tests.test_user_auth#test_user_login_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_auth"}, {"name": "host", "value": "MarvelDeviser"}, {"name": "thread", "value": "68-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_auth"}]}